man/tpm2_policysecret.1.md

```diff
@@ -0,0 +1,69 @@
+% tpm2_policysecret(1) tpm2-tools | General Commands Manual
+%
+% OCTOBER 2018
+
+# NAME
+
+**tpm2_policysecret**(1) - Enables secret(password/ hmac) based authorization to a policy.
+
+# SYNOPSIS
+
+**tpm2_policysecret** [*OPTIONS*]
+
+# DESCRIPTION
+
+**tpm2_policysecret**(1) Enables secret (password/ hmac) based authorization to a policy. The secret is the authentication value of any TPM object (NV/ Hierarchy/ Loaded/ Persistent Handles).
+
+# OPTIONS
+
+  * **-c**, **--context**=_OBJECT_CONTEXT_:
+
+    A context object specifier of a transient/ permanent/ persistent object. Either a file path of a object context blob or a loaded/ persistent/ permanent handle id. See section "Context Object Format".
+
+  * **-p**, **--auth-value**=_AUTH\_VALUE_:
+
+    The authentication value of the object whose secret will be referred in the policy. Authentication values should follow the "authorization formatting standards", see section "Authorization Formatting".
+
+  * **-S**, **--session**=_SESSION_FILE_:
+
+    The policy session file generated via the **-S** option to
+    **tpm2_startauthsession**(1).
+
+  * **-o**, **--policy-file**=_POLICY\_FILE_:
+
+    File to save the policy digest.
+
+[common options](common/options.md)
+
+[common tcti options](common/tcti.md)
+
+[supported hash algorithms](common/hash.md)
+
+[algorithm specifiers](common/alg.md)
+
+# EXAMPLES
+
+Choose a TPM handle like TPM_RH_OWNER. Supply the 32 bit handle # of the object to the **tpm2_policysecret** tool along with the authentication value for the object in a *trial* session. Use the resultant policy digest as 
+input authentication policy when creating a sealing object. In a *real* policy session supply the auth object's authentication data along with object's context/ handle id, if the secret provided is correct, unsealing should succeed.
```

Comment:
The first few sentences should explain what your accomplishing not how you're doing it. Please re-write this with the opening statement being what you want to accomplish, followed by how you will accomplish it.